src/main/java/com/minecolonies/coremod/entity/citizen/VisitorCitizen.java

```diff
@@ -688,4 +689,15 @@ protected void dropInventory()
             }
         }
     }
+
+    // TODO:REMOVE DEBUG
```

Comment:
todo, remove debug?

Reply:
ye its debug output to find the issue causing visitors dying at zero positions and is to be removed in the future xD